packages/framework-types/src/effect/index.ts

```diff
@@ -1,19 +1,849 @@
-// TODO: Document thoroughly
+/**
+ * effect-ts reexports
+ * ===================
+ *
+ * Module to work with the layered architecture of some Booster parts.
+ *
+ * This module reexports the minimum types and functions from the `effect-ts` library to
+ * be able to work effectively throughout the codebase.
+ *
+ * Keep reading these comments to get a grasp on what does all the stuff mean and how to
+ * work with these concepts.
+ *
+ *
+ */
+
+/*************************************************
+ *                                               *
+ *             INTRODUCTION                      *
+ *                                               *
+ *************************************************/
+
+/* What is effect-ts for?
+ * ----------------------
+ *
+ * `effect-ts` is a TypeScript port of the ZIO library, used for reactive programming in Scala.
+ * It is used in production, and has proven to reduce complexity in many projects. Currently we're using the
```

Comment:
```suggestion
 * It is used in production and has proven to reduce complexity in many projects. Currently, we're using the
```

Grammarly suggested this 🤷